fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -187,6 +184,14 @@ def self.run(options)
         end
       end
 
+      def self.puts_download_url(body)
+        version_url = body['public_url']
+        version_url = body['public_url'] + '/app_versions/' + body['config_url'].split('/').last if body['config_url']
```

Comment:
if `body` is a normal Hash returning nil if a key is missing, and nil behaves as normal, you will end up with
```
2.3.3 :001 > {}['hello'].split('/')
NoMethodError: undefined method `split' for nil:NilClass
```